apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/DatabaseTestSchemaManager.java

```diff
@@ -61,6 +66,17 @@ public static boolean createOrUpdateSchema(
         && flywayInfo.current().getState() == MigrationState.SUCCESS;
   }
 
+  /**
+   * Private method to handle Exceptions that are thrown.
+   *
+   * @param message Error message for exception.
+   * @param exception Exception that is being logged.
+   */
+  private static void handleException(String message, Exception exception) {
+    LOGGER.error(message, exception);
+    exception.printStackTrace();
```

Comment:
I know this line was requested, but I think we try and avoid printing stack traces for flyway and SQL exceptions to avoid spitting database info into the logs.